micro-frontends/src/next-ui/Containers/patientAlergies/PatientAlergiesControl.jsx

```diff
@@ -53,14 +58,24 @@ export function PatientAlergiesControl(props) {
     return reactions;
   };
 
+  const TransformSeverityData = (severityData) => {
+    const {setMembers, answers} = severityData;
+    const severities = setMembers.length > 0 ? setMembers: answers;
+    const extractedSeverity = [];
+    severities.forEach( severity => {
+      extractedSeverity.push({ name: severity.display, uuid: severity.uuid });
```

Comment:
can we use map method instead of forEach? The purpose of map method of array is same